src/interactive.js

```diff
@@ -150,9 +154,15 @@ module.exports = function (manifest, manifestPath, ugly) {
 
     function promptFieldValue(fieldName) {
         let value = '';
+        let currentValuePrompt = manifest[fieldName] ? '('.grey + manifest[fieldName].grey + ') '.grey : "";
+        let verb = manifest[fieldName] ? 'Edit' : 'Enter';
```

Comment:
@nicolayr Thanks for pointing out.
I have fixed in https://github.com/adhbh/d2-manifest/commit/bef97a5d2c1f50318a2b1172a9bf76e686cb5561 . 

However, there doesn't seem to be an option to edit the custom fields. I am opening a new issue for it.